tools/pysa_integration_tests/model_verify.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+import venv
+import os
+import shutil
+import tempfile
+from pathlib import Path
+from ...api.connection import PyreConnection
+from ...api.query import get_invalid_taint_models
+
+cur_dir = Path(__file__).parent
+envpath = cur_dir / ".env"
+requirements_path = cur_dir / "verify_models" / "requirements.txt"
+activate = envpath / "bin" / "activate"
+command = f"pip install -r {requirements_path}"
+
+if os.path.exists(envpath):
+  shutil.rmtree(envpath)
+os.mkdir(envpath)
+venv.create(envpath, with_pip=True)
+
+
+subprocess.check_call(["bash", "-c", f"source {activate}; {command}"])
+
+pyre_connection = PyreConnection(cur_dir / "verify_models")
+
+temp_file = tempfile.TemporaryFile(prefix=".pyre_configuration", suffix=".local", dir=cur_dir)
+temp_file.write(b'{"source_directories": ["verify_models"]}')
```

Comment:
Put in as well taint models path pointing to the repositories stubs? This helps us make the action verify newer models as well.